files/install_graphdb.sh

```diff
@@ -47,3 +47,6 @@ mv /tmp/graphdb.service /lib/systemd/system/graphdb.service
 systemctl daemon-reload
 systemctl enable graphdb.service
 systemctl start graphdb.service
+
+# Shred authorized_keys
+shred -u /root/.ssh/authorized_keys /home/ubuntu/.ssh/authorized_keys || true
```

Comment:
this can be removed
`ssh_clear_authorized_keys = true` in `azure.pkr.hcl` does the same.

Reply:
Done